src/components/standard-button/standard-button.component.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './standard-button.styles.scss';
+
+const StandardButton = (props) => (
+
+    <div>
+        <div style={{ height: props.height, width: props.width }} className="wrapper">
+            <div className="cta">
+                <span>{props.children}</span>
```

Comment:
not a big thing but I couldn't see a reason for the span.
try to use minimal html tags all the time.
If its not necessary, consider removing the span tag.